packages/next-templates/src/app/klachtenformulier-niet-ingelogd/stap2/page.tsx

```diff
@@ -0,0 +1,214 @@
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${stepProgressLabel}: ${stepLabel} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+      <PageContent className="voorbeeld-page-content-flex">
+        <BreadcrumbNav className="voorbeeld-breadcrumb-nav" label="Kruimelpad">
+          <BreadcrumbNavLink href="/klachtenformulier-niet-ingelogd/stap1">
+            <IconArrowLeft />
+            Vorige Stap
+          </BreadcrumbNavLink>
+        </BreadcrumbNav>
+
+        <Article id="main" className="voorbeeld-article-space voorbeeld-main-layout">
+          <div>
+            <Heading1>Klacht over de gemeente doorgeven </Heading1>
+
+            <HeadingGroup className="voorbeeld-headinggroup">
+              <PreHeading>{stepProgressLabel}</PreHeading>
+              <Heading2>{stepLabel}</Heading2>
+              <Paragraph>Vul de vakjes met een * altijd in. Anders kunt u niet verder.</Paragraph>
+            </HeadingGroup>
+
+            <form>
+              <Fieldset>
+                <FieldsetLegend>Bent u een persoon of een bedrijf?</FieldsetLegend>
+                <FormField type="radio">
+                  <Paragraph className="utrecht-form-field__label utrecht-form-field__label--radio">
+                    <FormLabel htmlFor="persoon" type="radio">
+                      <RadioButton
+                        name="persoon-of-bedrijf"
+                        id="persoon"
+                        className="utrecht-form-field__input"
+                      ></RadioButton>
+                      Persoon
+                    </FormLabel>
+                  </Paragraph>
+                </FormField>
+
+                <FormField type="radio">
+                  <Paragraph className="utrecht-form-field__label utrecht-form-field__label--radio">
+                    <FormLabel htmlFor="bedrijf" type="radio">
+                      <RadioButton
+                        name="persoon-of-bedrijf"
+                        id="bedrijf"
+                        className="utrecht-form-field__input"
+                      ></RadioButton>
+                      Bedrijf
+                    </FormLabel>
+                  </Paragraph>
+                </FormField>
+              </Fieldset>
+
+              {/* staat in figma niet in fieldset - wel gedaan want h3 per sectie in form - h3 in fieldsetlegend */}
+              <Fieldset>
+                <FieldsetLegend>
+                  <Heading3>Persoonsgegevens</Heading3>
+                </FieldsetLegend>
+                <FormField>
+                  <FormLabel htmlFor="voorletter">Voorletter(s)</FormLabel>
+                  <FormFieldDescription></FormFieldDescription>
+                  {/* zonder formfielddescription komt textbox naast formlabel te staan, met descrpt. eronder?  */}
+                  <Textbox id="voorletter"></Textbox>
```

Comment:
self closing tag. en kleinere text boxes laat de gebruiker weten hoe veel ze in de textbox moeten zetten. dus je kan er een class name op zetten en een max-width geven